src/dbaas/logs/order/order.lexi.service.js

```diff
@@ -6,7 +6,15 @@ angular.module("ovh-api-services").service("OvhApiDbaasLogsOrderLexi", function
     var orderResource = $resource("/order/upgrade/logs/:serviceName", {
```

Comment:
The file is not at the right place ;-) The folder structure follows the `$resource` urls. So this file should be located in /order folder

Reply:
So order will be a separate folder in dbaas and logs will come inside it right ?
Since there is only one api at present, i have not created upgrade or logs folder. Might have to create when we put more apis.